Airports/EGDI/Sector.txt

```diff
@@ -0,0 +1,7 @@
+CIRCLE_SECTORLINE:DITWR:EGDI:2
+DISPLAY:DITWR:DITWR:DIAPP
+DISPLAY:DIAPP:DITWR:DIAPP
+
+CIRCLE_SECTORLINE:DIAPP MATZ CIRCLE:EGDI:3
+DISPLAY:Swanwick Mil South West:Swanwick Mil South West:DIAPP
+DISPLAY:DIAPP:Swanwick Mil South West:DIAPP
```

Comment:
Again, when you are logged on as Yeovil and Swanwick Mil is above you, do you really need a ring around Merryfield _and_ Yeovil, or just around Yeovil?